en/docs/setup/installing-as-a-linux-service.md

```diff
@@ -39,9 +39,8 @@ product directory location.
     exit 1
     esac
     ```
-
-    For example, given below is a startup script written for WSO2
-    Identity Server 5.4.0:
+    Following is an example startup script written for WSO2
+    Identity Server 5.4.0
```

Comment:
5.9.0?

Reply:
The given example is a config for a 5.4.0. Therefore, I had to go with 5.4.0.